src/locale/properties/search/search.en.properties

```diff
@@ -14,3 +14,4 @@ ngOrcid.search.keyword=Keyword
 ngOrcid.search.otherNameFields=Also search other name fields
 ngOrcid.search.search=SEARCH
 ngOrcid.search.orcidIdPatterError=Given value doesn't match ORCID iD pattern
+ngOrcid.search.ariaLabelSearchResults=Search Results
```

Comment:
properties missing in xx, ltr and rtl properties files

Reply:
oops cloned some missing things, thanks  👍